src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -57,7 +57,6 @@ public class ConfigurationManager<T> : BaseConfigurationManager, IConfigurationM
         /// </summary>
         static ConfigurationManager()
         {
-            LogHelper.LogVerbose("Assembly version info: " + LogHelper.MarkAsNonPII(typeof(ConfigurationManager<T>).AssemblyQualifiedName));
```

Comment:
Did we move this somewhere else? Or do we say: not needed? ? Is it line 438 of LogHelper?